src/components/pages/Categories/CategoryList.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="articles">
+    <app-Category-Post
+      class="form"
+      :loading="loading"
+      :access="access"
+    />
+    <app-Category-List
```

Comment:
[must]
コンポーネントのタグはケバブケースで良いので、

```suggestion
    <app-category-list
```

に修正をお願いします！
他にも同様の箇所は合わせて修正をお願いします！